django/core/cache/backends/memcached.py

```diff
@@ -177,3 +179,14 @@ def close(self, **kwargs):
         # libmemcached manages its own connections. Don't call disconnect_all()
         # as it resets the failover state and creates unnecessary reconnects.
         pass
+
+    def set(self, key, value, timeout=DEFAULT_TIMEOUT, version=None):
+        """PyLibMC throws an exception if the stored value is too large,
+python-memcache instead returns False. We need to override set here to account
+for the difference."""
+        key = self.make_key(key, version=version)
+        try:
+            self._cache.set(key, value, self.get_backend_timeout(timeout))
+        except self.too_big_exception:
```

Comment:
Can this just be `except self._lib.TooBig:` to avoid the assignment in `__init__()`?